translations/search/students/absenceHistoryDialog/en-US.json

```diff
@@ -2,6 +2,7 @@
   "history": {
     "title": "Absence history",
     "item": {
+      "null": "Unknown reason",
```

Comment:
I think “No data” would be more appropriate. Pretty sure that’s the phrase used elsewhere in the app as well.

Reply:
Good idea.